docs/standards/smirnoff.md

```diff
@@ -385,7 +390,7 @@ separate methods must be specified for periodic (`periodic_potential`) and non-p
 
 The `periodic_potential` attribute specifies the manner in which electrostatic interactions are to be computed in periodic systems. Allowed values are:
 
-* `Ewald3D-ConductingBoundary` (default) denotes that a method like [particle mesh Ewald](https://docs.openmm.org/latest/userguide/theory.html#coulomb-interaction-with-particle-mesh-ewald) should be used
+* `Ewald3D-ConductingBoundary` (default) denotes that a method like [particle mesh Ewald](http://docs.openmm.org/development/userguide/theory/02_standard_forces.html#coulomb-interaction-with-ewald-summation) should be used.
```

Comment:
Should this URL be a fixed version link, or the `development` version link of the user guide? Both `latest/` and `development/` change every release, but `http://docs.openmm.org/8.0.0/userguide/theory/02_standard_forces.html#coulomb-interaction-with-ewald-summation` will be constant.

Reply:
A fixed version is probably better - I'll update the diff here to be consistent, but we should also look to unify this across the rest of the spec. I see many "unstable" links elsewhere in the spec.